docker/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:14.04
+#QIIME basic installation env v0.1 
+MAINTAINER Chester Kuo <chester.kuo@gmail.com>
+
+RUN apt-get update && apt-get -y install curl
+RUN apt-get -y --no-install-recommends install cmake mercurial git make subversion
+RUN apt-get -y install gcc g++ gdb
+RUN apt-get -y install python python-dev python-pip vim
+RUN apt-get -y install pkg-config libpng-dev libfreetype6-dev
+RUN apt-get -y install libopenblas-base libopenblas-dev libatlas-base-dev gfortran libblas-dev liblapack-dev mklibs
+
+RUN mkdir -p /tmp
```

Comment:
Sounds like a reasonable suggestion, but I guess the real location where
we would want the temp folder, is wherever the EC2 instance running this
docker image has some scratch space, right?. I'll have to check the
documentation and see if I can find something about this.

On (May-27-15|16:46), Greg Caporaso wrote:
> > @@ -0,0 +1,21 @@
> > +FROM ubuntu:14.04
> > +#QIIME basic installation env v0.1 
> > +MAINTAINER Chester Kuo <chester.kuo@gmail.com>
> > +
> > +RUN apt-get update && apt-get -y install curl
> > +RUN apt-get -y --no-install-recommends install cmake mercurial git make subversion
> > +RUN apt-get -y install gcc g++ gdb
> > +RUN apt-get -y install python python-dev python-pip vim
> > +RUN apt-get -y install pkg-config libpng-dev libfreetype6-dev
> > +RUN apt-get -y install libopenblas-base libopenblas-dev libatlas-base-dev gfortran libblas-dev liblapack-dev mklibs
> > +
> > +RUN mkdir -p /tmp
> 
> Maybe a different directory? For example, ``$HOME/temp``, and then set that in the qiime_config, which you could do with:
> 
> ```bash
> echo "temp_dir $HOME/temp" > $HOME/.qiime_config
> ```
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/basespace-qiime/pull/2/files#r31192078